submarine-server/server-core/src/test/java/org/apache/submarine/server/rest/JobManagerRestApiTest.java

```diff
@@ -156,8 +149,7 @@ public void testDeleteJobById() throws IOException {
 
     Gson gson = new Gson();
     JsonResponse jsonResponse = gson.fromJson(requestBody, JsonResponse.class);
-    assertEquals("Response code should be 200 ",
-        Response.Status.OK.getStatusCode(), jsonResponse.getCode());
+    assertEquals(Response.Status.NOT_FOUND.getStatusCode(), jsonResponse.getCode());
```

Comment:
`testDeleteJobById()` needs to be able to real delete the JOB  instead of `NOT_FOUND` JOB, otherwise this test case is meaningless.